src/pipecat/services/azure.py

```diff
@@ -162,7 +162,7 @@ async def cancel(self, frame: CancelFrame):
 
     def _on_handle_recognized(self, event):
         if event.result.reason == ResultReason.RecognizedSpeech and len(event.result.text) > 0:
-            frame = TranscriptionFrame(event.result.text, "", int(time.time_ns() / 1000000))
+            frame = TranscriptionFrame(event.result.text, "", str(int(time.time_ns() / 1000000)))
```

Comment:
Nice catch! I forgot to update that one. Can you use `time_now_iso8601()` instead?

```
from pipecat.utils.time import time_now_iso8601
```

Reply:
Done